src/config.rs

```diff
@@ -623,17 +622,14 @@ mod tests {
             .file,
             FileTarget::ComplexTemplate(PathBuf::from("~/.QuarticCat").into()),
         );
-        assert_eq!(
-            parse(
-                r#"
-                    [file]
-                    target = '~/.QuarticCat'
-                    type = 'symbolic'
-                    append = 'whatever'
-                "#,
-            )
-            .is_err(),
-            true
-        );
+        assert!(parse(
```

Comment:
can even do `parse(...).unwrap_err()` instead of assert

Reply:
Yep. It also includes the `Ok` value in the panic message, which is nice.